flink-runtime/src/main/java/org/apache/flink/runtime/checkpoint/CheckpointCoordinator.java

```diff
@@ -1935,6 +1935,7 @@ private void abortPendingCheckpoint(
                 }
             } finally {
                 sendAbortedMessages(
+                        pendingCheckpoint.getTasksToCommitTo(),
```

Comment:
Why don't we need to send it to the `tasksToWaitFor`? Or is it that `tasksToCommitTo` is a strict superset of `tasksToWaitFor`?

Reply:
Based on current implementation `tasksToWaitFor` should be always equivalent to `tasksToCommitTo`. For me using `tasksToCommitTo` mainly comes from to keep the logic un-change, and also be symmetric with `sendCommitMessages`.